### PR TITLE
Add license.

### LINKED ISSUE #35: What license does this project use?
Hey!

This project currently does not have a license, meaning it's a bit risky to both use and extend. (No license generally implies proprietary!).

We are doing some work to build a testing framework like sol-rs in [PrimaBlock/parables](https://github.com/PrimaBlock/parables). Most of it is a re-implementation, but we have a handful of things that are copied ([wei.rs](https://github.com/PrimaBlock/parables/blob/master/testing/wei.rs) and the stack management in [trace.rs](https://github.com/PrimaBlock/parables/blob/master/testing/trace.rs#L178)). We want to make sure to give proper attribution. Your project showed that this kind of testing was possible and broke a ton of ground for us.

Could you make sure to add the appropriate license files?

### PR DESCRIPTION
**Pull request by tomusdrw**
Closes #35 

- [x] TODO add preambles to all files

### CODE REVIEW

**Comment by ddorgan** [comment]
[clabot: test]

**Comment by ddorgan** [comment]
[clabot: test]

**Comment by ddorgan** [comment]
[clabot:check]

**Comment by 5chdn** [comment]
can we please merge this?

**Comment by snd** [comment]
@5chdn i don't think this is complete until the preambles are added to the files (see TODO)

**Comment by tomusdrw** [comment]
Sorry forgot about this one. Preambles added (assuming this file is part of Parity).

File: license_header [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
```
Comment (snd) [inline license_header]: should this read "This file is part of Parity sol-rs"? also wherever "Parity" is mentioned below

**Pull request merged**